### ISSUE TITLE
Deactivation confirmation modal window error

### ISSUE THREAD

**Issue by mailanand75** [JodiLogik/phase_one#66]
In the below screenshot, 
- remove the "In the meanwhile...." This was supposed to have been removed across the board.
  -resize the modal window after removing "in the meanwhile..." 
- Increase the width of the modal window so that "you" doesnt stand out as a separate line.

![image](https://cloud.githubusercontent.com/assets/12179089/15673533/f6cc243c-2754-11e6-8994-0a5b5d1586c8.png)

**Comment by cyrus-gc** [comment]
Fixed

**Comment by mailanand75** [comment]
Sorry, noticed one more issue which I could have pointed out the first time I reported the issue:

**Replace "Can't find an email after 24 hours? Check your spam folder." with 
"Can't find our confirmation email? Check the spam folder in your email inbox."**

Waiting for 24 hours is applicable only when I used to manually approve news users.

![image](https://cloud.githubusercontent.com/assets/12179089/15735575/78f81386-28b7-11e6-9de4-dd1429c3544b.png)
